Library/Homebrew/sorbet/tapioca/config.yml

```diff
@@ -0,0 +1,44 @@
+---
+# see https://github.com/Shopify/tapioca?tab=readme-ov-file#configuration
+gem:
+  exclude:
+    # These conflict with the rbi files provided by sorbet:
+    - json
+    - msgpack
+    # These aren't needed:
```

Comment:
> doesn't noticeably speed up typecheck updates

Given this, I think this will just result in more work needed to keep this exclude list updated and correct.

Reply:
Maybe there's some way to automatically audit this to make sure it's up-to-date. Removing 60k lines from the codebase seems useful too.

Reply:
I added a test that validates the list of exclusions are all actual dependencies. (Though, I don't feel too strongly about greatly expanding the list here, should folks still have reservations.)

Reply:
I can imagine a `brew typecheck` feature that can identify an incorrect `exclude`, based on the namespace of a type error, but that's probably more trouble than it's worth.

I've asked in the #tapioca channel of the sorbet slack to see if excluding unreferenced dependencies is something other folks bother with, I will update if I hear anything back.

Reply:
I use this list on another project and: it becomes pretty obvious when things are needed because `brew typecheck` will fail until RBI files are added.